From: Maximiliano Curia <maxy@gnuservers.com.ar>
Date: Tue, 6 Dec 2016 18:53:58 +0100
Subject: Delete the properties window if closed

This should allow the window to be created again on request.
---
 src/widgets/collectionpropertiesdialog.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/widgets/collectionpropertiesdialog.cpp b/src/widgets/collectionpropertiesdialog.cpp
index 8e31957..6922306 100644
--- a/src/widgets/collectionpropertiesdialog.cpp
+++ b/src/widgets/collectionpropertiesdialog.cpp
@@ -119,6 +119,7 @@ void CollectionPropertiesDialog::Private::init()
 {
     QVBoxLayout *mainLayout = new QVBoxLayout;
     q->setLayout(mainLayout);
+    q->setAttribute(Qt::WA_DeleteOnClose);
     mTabWidget = new QTabWidget(q);
     mainLayout->addWidget(mTabWidget);
 
